src/parsers/manifest/types.ts

```diff
@@ -110,6 +110,10 @@ export interface IParsedRepresentation {
    * Not set if unknown or if it makes no sense (e.g. for audio).
    */
   width?: number;
+  /**
+   * Information about the HDR characteristic of a content.
+   */
+  hdr?: IHDRInformation;
```

Comment:
Nice.

Maybe something hdrInfo may be more indicatives of that format's data, no?